src/docs/product/integrations/deployment/vercel/index.mdx

```diff
@@ -54,7 +54,7 @@ Use Vercel to [link projects](#project-linking) for uploading source maps and no
 
 - If you have not already done so, [instrument your code with Sentry](/platforms/javascript/).
 - Ensure you have [installed a repository integration](/product/releases/setup/release-automation/) and added the relevant repository.
-- Add the [Sentry Webpack Plugin](https://github.com/getsentry/sentry-webpack-plugin) to your configuration. For NextJS, use `next.config.js`.
+- Add the [Sentry Webpack Plugin](https://github.com/getsentry/sentry-webpack-plugin) to your Webpack configuration. For Next.js, if you followed the `@sentry/nextjs` [set-up guide](platforms/javascript/guides/nextjs/), this will already have been done.
```

Comment:
Thanks to @kamilogorek for catching this.

```suggestion
- Add the [Sentry Webpack Plugin](https://github.com/getsentry/sentry-webpack-plugin) to your Webpack configuration. For Next.js apps, if you followed the `@sentry/nextjs` [set-up guide](/platforms/javascript/guides/nextjs/), this will already have been done.
```